lib/custom_a11y_rules.js

```diff
@@ -156,6 +171,26 @@ var customRules = {
             evaluate: function(node, options) {
                 return axe.utils.isHidden(node);
             },
-        }
+        },
+        {
+            'id': 'section-heading-first-child',
+            'metadata': {
+                'impact': 'minor',
+                'messages': {
+                    'pass': '<section> elements should have a heading as its first child',
+                    'fail': '<section> elements should have a heading as its first child'
+                }
+            },
+            evaluate: function(node, options) {
+                var childNode = node.children[0].nodeName,
+                    validHeadings = ['H1', 'H2', 'H3', 'H4', 'H5', 'H6'];
```

Comment:
What would we encounter on DOMs? My point is we should make the assertion case-insensitive, shouldn't we?